sonar-db/src/main/java/org/sonar/db/component/SnapshotDao.java

```diff
@@ -63,6 +64,10 @@ public SnapshotDto selectLastSnapshotByComponentId(DbSession session, long compo
     return mapper(session).selectLastSnapshot(componentId);
   }
 
+  public List<SnapshotDto> selectLastSnapshotByComponentIds(DbSession dbSession, List<Long> componentIds) {
```

Comment:
I think that you'll have to update this by using component uuid (Seb has committed I think)